<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UIResponsable</class>
 <widget class="QMainWindow" name="UIResponsable">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1270</width>
    <height>690</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Interface Resonsable</string>
  </property>
  <property name="windowIcon">
   <iconset resource="src.qrc">
    <normaloff>:/image/responsable.png</normaloff>:/image/responsable.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">#centralwidget{
 border-image:url(:/image/bg.jpg);

}

#widget{
 border: 1px solid;
background: rgba(0, 0, 0, 80);

}

#label,#label_2,#currentDateTime,#logo,#label_3,#label_5,#label_15,#labelRechercher{ color: white; }
QLineEdit,QDateEdit ,#comboBoxRole{ border-radius: 3px; }
#label_13,#label_12,#label_6,#label_8,#label_7,#label_9,#label_10,#label_14,#label_11,#label_4,#labelRole,#labelPassword,#labelConfirmPassword{

color: white;
 
  font-family: Arial;

}



QPushButton, #comboBoxRechercherPar
{
  color: white;
  background-color: #27a9e3;
  border-width: 0px;
  border-radius: 3px;
}

QPushButton:hover { background-color: #66c011; }

#topPanel { background-color: qlineargradient(spread:reflect, x1:0.5, y1:0, x2:0, y2:0, stop:0 rgba(46, 204, 233, 100), stop:1 rgba(32, 80, 96, 5)); 


}


#restartButton{
 border-radius: 15px;

}
#shutdownButton{
 border-radius: 15px;

}

QGroupBox{

color: white;

}



#radioButtonCreer,#radioButtonModifier{

 color: white;

}

#labelUtilisateurConnecteTitre{

color: white;
 
  font-family: Arial;

}

#labelUtilisateurConnecte{

color: white;
 
  font-family: Arial;
}










</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>50</y>
      <width>521</width>
      <height>141</height>
     </rect>
    </property>
    <property name="title">
     <string>AJOUT  UNE CLASSE  &amp; Module</string>
    </property>
    <widget class="QGroupBox" name="groupBox_5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>241</width>
       <height>111</height>
      </rect>
     </property>
     <property name="title">
      <string>CLASSE</string>
     </property>
     <widget class="QLineEdit" name="lineEdit_classe">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>20</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>101</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Nom de la classe :</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_capacite">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>50</y>
        <width>41</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>121</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>capacité de la classe :</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_Classe">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>80</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Engister</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_6">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>20</y>
       <width>251</width>
       <height>111</height>
      </rect>
     </property>
     <property name="title">
      <string>Module</string>
     </property>
     <widget class="QLineEdit" name="lineEdit_Module">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>50</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_12">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>47</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string>Module :</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox_classe">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>151</width>
        <height>22</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">#comboBox_classe
{
  color: white;
  background-color: #27a9e3;
  border-width: 0px;
  border-radius: 3px;
}

#comboBox_classe:hover { background-color: #66c011; }</string>
      </property>
      <item>
       <property name="text">
        <string>Liste Classe</string>
       </property>
      </item>
     </widget>
     <widget class="QLineEdit" name="lineEdit_Matiere">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>80</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_14">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>80</y>
        <width>47</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string>  Matière : </string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_Module">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>80</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Engister</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>0</y>
      <width>1211</width>
      <height>47</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QPushButton" name="pushButtonProfil">
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>35</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Consulter mon profil ...</string>
       </property>
       <property name="text">
        <string>Profile</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_gerer">
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>35</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Consulter mon profil ...</string>
       </property>
       <property name="text">
        <string>   Gerer les Formateur  </string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="labelRechercher">
       <property name="text">
        <string>Rechercher un utilisateur à partir de son</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBoxRechercherPar">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>30</height>
        </size>
       </property>
       <property name="currentIndex">
        <number>0</number>
       </property>
       <item>
        <property name="text">
         <string>Id</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Username</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEditRechercher">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>30</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonRechercher">
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>35</height>
        </size>
       </property>
       <property name="text">
        <string>Rechercher</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>200</y>
      <width>521</width>
      <height>431</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="title">
     <string> ENREGISTREMENT D'UN  FORMATEUR   Etudiant</string>
    </property>
    <widget class="QGroupBox" name="groupBox_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>200</y>
       <width>381</width>
       <height>141</height>
      </rect>
     </property>
     <property name="title">
      <string>INFORMATION PROFESSOINNELLES</string>
     </property>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>31</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Tel :</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBoxRole">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>70</y>
        <width>111</width>
        <height>30</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>30</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">#comboBoxRole
{
  color: white;
  background-color: #27a9e3;
  border-width: 0px;
  border-radius: 3px;
}

#comboBoxRole:hover { background-color: #66c011; }

</string>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <item>
       <property name="text">
        <string>FORMATEUR</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>ETUDIANT</string>
       </property>
      </item>
     </widget>
     <widget class="QLabel" name="labelRole">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>70</y>
        <width>27</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>Rôle :</string>
      </property>
     </widget>
     <widget class="QWidget" name="formLayoutWidget">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>20</y>
        <width>211</width>
        <height>124</height>
       </rect>
      </property>
      <layout class="QFormLayout" name="formLayout">
       <item row="0" column="1">
        <widget class="QLineEdit" name="lineEditLogin">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>30</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="labelPassword">
         <property name="text">
          <string>Password :</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="lineEditPassword">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>30</height>
          </size>
         </property>
         <property name="echoMode">
          <enum>QLineEdit::Password</enum>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="labelConfirmPassword">
         <property name="text">
          <string>Confirm password :</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="lineEditConfirmPassword">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>30</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="echoMode">
          <enum>QLineEdit::Password</enum>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label_11">
         <property name="text">
          <string>Login :</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QLineEdit" name="lineEdit_Tel">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>30</y>
        <width>113</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QPushButton" name="pushButton_3">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>400</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Effacer</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_valider">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>400</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Valider</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>210</y>
       <width>111</width>
       <height>91</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">#label_10{ border: 1px solid;
      border-image:url(:/image/u.jpg);

}</string>
     </property>
     <property name="text">
      <string>PHOTO</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_5">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>310</y>
       <width>111</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Inserer la photo</string>
     </property>
    </widget>
    <widget class="QWidget" name="horizontalLayoutWidget_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>350</y>
       <width>250</width>
       <height>39</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QRadioButton" name="radioButtonCreer">
        <property name="text">
         <string>Creer</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioButtonModifier">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>Modifier</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>501</width>
       <height>181</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="title">
      <string>INFORMATION SUR USER</string>
     </property>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>60</y>
        <width>51</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Noms :</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>100</y>
        <width>71</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Numero CNI : </string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_CNI">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>100</y>
        <width>131</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEditUsername">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>60</y>
        <width>131</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>60</y>
        <width>61</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Prenoms :</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEditprenome">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>60</y>
        <width>151</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>140</y>
        <width>61</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Date Naiss :</string>
      </property>
     </widget>
     <widget class="QDateEdit" name="dateEdit_Naiss">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>140</y>
        <width>131</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>100</y>
        <width>61</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Lieu Naiss :</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_lieu">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>100</y>
        <width>151</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>140</y>
        <width>41</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Sexe :</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBoxRole_Sexe">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>140</y>
        <width>151</width>
        <height>30</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>30</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">#comboBoxRole_Sexe
{
  color: white;
  background-color: #27a9e3;
  border-width: 0px;
  border-radius: 3px;
}

#comboBoxRole_Sexe:hover { background-color: #66c011; }

</string>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <item>
       <property name="text">
        <string>Homme</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Femme</string>
       </property>
      </item>
     </widget>
     <widget class="QLabel" name="label_13">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>47</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Id :</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEditId">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>20</y>
        <width>131</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_Matricule">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>20</y>
        <width>151</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_15">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>20</y>
        <width>61</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Matricule :</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QTableView" name="tableView">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>50</y>
      <width>671</width>
      <height>501</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>580</y>
      <width>671</width>
      <height>41</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_Etudiant">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>35</height>
        </size>
       </property>
       <property name="text">
        <string>   Liste Etudient   </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_Formateur">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>35</height>
        </size>
       </property>
       <property name="text">
        <string>   Liste Formateur  </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_Matiere">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>35</height>
        </size>
       </property>
       <property name="text">
        <string>  Liste  Matière  </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonEffacer">
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>35</height>
        </size>
       </property>
       <property name="text">
        <string>Effacer</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonExit">
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>35</height>
        </size>
       </property>
       <property name="text">
        <string>Exit</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_4">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>640</y>
      <width>221</width>
      <height>32</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="labelUtilisateurConnecteTitre">
       <property name="text">
        <string>Utilisateur Connecté :</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelUtilisateurConnecte">
       <property name="text">
        <string>&lt;dynamic&gt;</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>560</y>
      <width>671</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="src.qrc"/>
 </resources>
 <connections/>
</ui>
